core/src/main/java/io/confluent/kafka/schemaregistry/storage/SchemaIdAndSubjects.java

```diff
@@ -53,6 +57,18 @@ public int getSchemaId() {
     return this.id;
   }
 
+  public boolean isSubjectVersionDeleted(String subject) {
```

Comment:
The method name is misleading since there would be only version per schema for a given subject.  Let me think of something more meaningful..